datadog-monitors-monitor-handler/docs/state-groups-.*.md

```diff
@@ -0,0 +1,80 @@
+# Datadog::Monitors::Monitor State Groups .*
```

Comment:
Is this file still supposed to be here after the schema update? (I may be misreading but I think this is the one that got removed)

Reply:
Good question, I'll check. We should probably in time setup a CI that would check for this.

Reply:
We actually removed `MonitorStateGroupValue`, but this is for `MonitorStateGroup`, so it's ok that it's here.

Reply:
Ahh ok cool. Thanks for checking!